explorer/client/@types/link-stats/index.d.ts

```diff
@@ -25,6 +25,8 @@ interface ITaskRun {
   status: string
   transactionHash?: string
   transactionStatus?: string
+  confirmations?: number
+  minimumConfirmations?: number
```

Comment:
Would minConfirmations be just as good of a name and save 4n characters 💭

Reply:
I prefer the name consistency with what's in core and in the receiving json from core. But I don't feel strongly about it.